reviews.Rmd

```diff
@@ -0,0 +1,583 @@
+The authors say they provide a 'Bayesian rationalization'.
+This is not correct.
+:::
+
+We respectfully disagree.
+What makes the arguments "Bayesian" is our interpretation of probabilities.
+For a Bayesian, they represent degrees of belief.
+For a frequentist, they represent (relative) frequencies of events that result from a random process.
+This is a subtle but most important distinction.
+
+:::reviewer
+Their rationalization is in essence based on the likelihood ratio.
+All else equal, a lower Type 1 error rate means that observing a significant result is relatively more likely when H1 is true.
+Of course a higher likelihood ratio also means a higher Bayes factor, which means a higher probability that H1 is true in a Bayesian framework.
+:::
+
+It is well known that the math for frequentists is similar or even identical to Bayesian under the right conditions.
+Still, a frequentist would not conclude anything about the probability of H1.
+However, our goal is to say something about how preregistration changes how believable researchers should find claims made in the literature.
+This necessitates a Bayesian perspective on the process of scientific inquiry (which we adopted) yet it applies to cases of researchers either using frequentist and Bayesian inference in their daily routine.
+
+:::reviewer
+But the workhorse in the rationalization that the authors provide is the likelihood ratio.
+This does not change when we put a distribution on the Type 1 error rate (as the authors do).
+:::
+
+Yes, the Bayes formula can be conceptualized as a likelihood ratio scaled by priors.
+However, the mere use of the likelihood ratio in the Bayes theorem can not be grounds to reduce Bayesianism to likelihood ratios.
+
+:::reviewer
+The distribution (or the move to a Bayesian rationalization) does not add anything (although of course it can be added).
+:::
+
+We find that it adds value by making an otherwise implicit process (that is, the process of updating beliefs in the theories) explicit to the point of making it calculable.
+To our knowledge, there is no other perspective in the literature that makes this part of the process explicit.
+By conceptualising uncertainty with a specific distribution, we rid ourselves of vague notions of transparency, control, or exploration.
+This is of utmost importance because only this clarity enables us to effectively navigate the tradeoffs involved.
+
+:::reviewer
+Importantly, the likelihood function of course also underlies the uniformly most powerful test in a Neyman-Pearson hypothesis testing framework.
+So for a frequentist (and I mean a frequentist not in statistical terms, but also in Popperian anti-Bayesian view on confirmation) the same argument for preregistration plays a role.
+:::
+
+We agree that this argument could perhaps be made for a frequentist as well.
```

Comment:
Important points, but maybe better for the high-level summary. So for this point, I would just add a short answer like "Yes, you could probably make a similar argument, but 1. we are not concerned with that 2. specific tradeoffs are hard". Then, in the high-level summary you could write about this point "Faced with a specific tradeoff, e.g., is a particular post hoc change of a preregistration beneficial, we must revert to vague notions and "gut" instinct under this philosophy.
Within the Bayesian framework, we may directly ask: What consequences should preregistration have on the beliefs of individual scientists or the research community as a whole?".